docs/pages/project/contributing/contributing-docs.md

```diff
@@ -162,7 +162,7 @@ bundle install{% endcapture %}
 
 If you've Docker and `make` installed in your system, then you can serve the site locally
 
-{% capture code_content %}make docker{% endcapture %}
+{% capture code_content %}make docker-build{% endcapture %}
```

Comment:
```suggestion
{% capture code_content %}make docker{% endcapture %}
```
Here  it's expected to run the `make docker` command inside `docs` dir.

Reply:
@vishalvivekm Yes. I am making some more changes to this file (mostly verbatim) to make instructions more clear. For example, add above this command that this should be run inside the docs folder.

Additional notes:
- The following statement needs clarification, "**_Alternatively, if you have Docker installed, then type `make docs-docker` to view the changes._**". The command essentially runs the docs site, similar to `make docker` command, just that this command would work from outside the _docs_ folder. It's not clear how this resolves the error.